persistence/src/commonMain/kotlin/com/wire/kalium/persistence/dao/message/MessageDAO.kt

```diff
@@ -11,7 +11,8 @@ data class MessageEntity(
     val senderUserId: QualifiedIDEntity,
     val senderClientId: String,
     val status: Status,
-    val visibility: Visibility = Visibility.VISIBLE
+    val visibility: Visibility = Visibility.VISIBLE,
+    val shouldNotify: Boolean = true
```

Comment:
Does this work as expected? Can't we remove this and compute it on fly based on if the sender the self user or not?

Reply:
ou, actually this is something that i forgot to remove! 

It was used for the prev. implementation